packages/tse-backend/src/modules/health/health.service.ts

```diff
@@ -0,0 +1,98 @@
+import { Injectable } from '@nestjs/common';
+import { CustomLogger } from '../logger/logger.service';
+import { InjectEntityManager } from '@nestjs/typeorm';
+import { DataSource, EntityManager } from 'typeorm';
+import { buildExchangeConfigs } from '../../common/utils/config-utils';
+import { ConfigService } from '@nestjs/config';
+import { CcxtGateway } from '../../integrations/ccxt.gateway';
+
+@Injectable()
+export class HealthService {
+  private readonly logger = new CustomLogger(HealthService.name);
+
+  constructor(
+    @InjectEntityManager() private entityManager: EntityManager,
+    private readonly dataSource: DataSource,
+    private readonly configService: ConfigService,
+    private readonly ccxtGateway: CcxtGateway,
+  ) {}
+
+  async geHealthStatuses() {
+    return {
+      db: await this.checkDbHealth(),
+      exchanges: await this.checkExchanges(),
```

Comment:
Maybe add number of strategies running?

Reply:
In commit 17cc74a0, I added a TODO annotation. I will implement this service after the pull requests related to the strategies have been merged